src/ntrip_ping.c

```diff
@@ -21,8 +26,11 @@ void usage(char *command)
   puts("\t--lat     <latitude>");
   puts("\t--lon     <longitude>");
   puts("\t--height  <height>");
-  puts("\t--epoch    <epoch>");
+  puts("\t--epoch   <epoch>");
   puts("\t--client  <X-SwiftNav-Client-Id header>");
+  puts("\t--no-eph  Disable ephemerides on connection (X-No-Ephemerides-On-Connection)");
+  puts("\t--verbose Enable curl verbose output");
+  puts("\t--debug   Enable debug output");
```

Comment:
consider updating the language here to be 
"Enable debug output [also enables verbose output]"
or similar

Reply:
The debug output actually replaces the standard verbose output.
`curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);` is required to use `curl_easy_setopt(curl, CURLOPT_DEBUGFUNCTION, debug_callback);`

Reply:
And `--debug` overwrites `--verbose`. So if you have `--verbose --debug`, it is as if you only have `--debug`

Reply:
Ah ok, I was just looking at the logic in the if section later were it said if verbose or debug then ...

Reply:
Yes, you are correct:
- For `--verbose`, you need `curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L)`
- For `--debug`, you need `curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L)` and `curl_easy_setopt(curl, CURLOPT_DEBUGFUNCTION, debug_callback)`
  - And I guess `curl_easy_setopt(curl, CURLOPT_DEBUGFUNCTION, debug_callback)` overwrites the default curl callback function that generates the verbose output, so you only end up with the debug output without the default verbose output